.buildkite/pipelines/emergency_release.yml

```diff
@@ -1,3 +1,18 @@
+env:
+  VERSION: ${BUILDKITE_COMMIT:0:12}
+  DOCKER_IMAGE_TAG: git-${VERSION}
+  DOCKER_IMAGE: docker.elastic.co/kibana-ci/kibana-serverless
+  EMERGENCY_RELEASE_REMOTE_SERVICE_CONFIG: https://raw.githubusercontent.com/elastic/serverless-gitops/main/gen/gpctl/kibana/emergency.yaml
+
 steps:
-  - label: Placeholder
-    command: echo "Hello!"
+  - label: ':label: Create tag'
+    command: .buildkite/scripts/serverless/create_deploy_tag/create_fix_tag.sh
+    if: build.branch =~ /^deploy-fix@.*\$/
+
```

Comment:
I've probably misunderstood something, but I would expect there to be a `wait` statement between the two steps so they don't run in parallel?

Reply:
Thanks, I pushed https://github.com/elastic/kibana/pull/176258/commits/14cd814f5035c30cb3ebecbdf0fdf7ba38786241